internal/minikube/minikube.go

```diff
@@ -17,16 +19,25 @@ const (
 
 //RunCmd executes a minikube command with given arguments
 func RunCmd(verbose bool, profile string, rawArgs ...string) (string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
```

Comment:
Will this timeout work with `minikube start`?
Maybe we should use the timeout that the cli command has?